PendingReleaseNotes.md

```diff
@@ -5,7 +5,9 @@
 - The removal of `CSI_ENABLE_READ_AFFINITY` option and its replacement with per-cluster
 read affinity setting in cephCluster CR (CSIDriverOptions section) in [PR](https://github.com/rook/rook/pull/13665)
 - Allow setting the Ceph `application` on a pool
-
+- updating `netNamespaceFilePath` for all clusterIDs in rook-ceph-csi-config configMap in [PR](https://github.com/rook/rook/pull/13613)
+  - Issue: The netNamespaceFilePath isn't updated in the CSI config map for all the clusterIDs when `CSI_ENABLE_HOST_NETWORK` is set to false in `operator.yaml`
+  - Impact: This results in the unintended network configurations, with pods using the host networking instead of pod networking.
```

Comment:
I need help understanding this pending release note. This is listed as a breaking change, but there are no instructions for users to follow to keep their cluster from breaking. Is this actually a breaking change?

@Madhu-1 @iPraveenParihar @travisn

Reply:
i think nothing was required from the user point and we never supported moving from multus/pod network to host networking or vice-versa, @iPraveenParihar can you please check this and confirm?

Reply:
@BlaineEXE actually this was an issue that I believed users should be aware of if they were using Pod networking.

Reply:
> i think nothing was required from the user point and we never supported moving from multus/pod network to host networking or vice-versa

I thought there was a period of time where we had `CSI_ENABLE_HOST_NETWORK: false` as the default value for all Rook users? Am I misremembering? 

If that was the default, then we need to have supported users setting that value back to `true` so they could get in line with the latest recommendations.

Reply:
It was enabled briefly three years ago (see #8085), but I'm not sure we need to worry about that too much since it was for such a short time and users would have had serious issues if they didn't already convert back to host networking.